test/envoye2e/driver/otel.go

```diff
@@ -114,6 +114,7 @@ func (v *verify) Run(p *Params) error {
 		return fmt.Errorf("timed out waiting for all metrics to match")
 	}
 }
+
```

Comment:
revert?

Reply:
They were added by make lint; I'll try and revert